izuna_ytdl/models/item.py

```diff
@@ -1,62 +1,29 @@
 import datetime
-from redis_om import JsonModel, Field, NotFoundError
-from typing import Optional, cast
-import logging
+from typing import List, TYPE_CHECKING
+import uuid as uuid_pkg
+from sqlmodel import Field, SQLModel, Relationship
 
+if TYPE_CHECKING:
+    from .download_task import DownloadTask
 
-class Item(JsonModel):
-    id: str = Field(index=True)
-    name: str
```

Comment:
The new schema seems like missing the item name/title, which is quite needed for item "adoption" later on.
Add this altogether with the import item script